src/lib/layout/unauthenticated.svelte

```diff
@@ -359,4 +366,13 @@
             padding-block-start: 5rem;
         }
     }
+
+    .mobile-logo {
+        position: fixed;
+        bottom: 1.25rem;
+        left: 50%;
+        transform: translateX(-50%);
+        z-index: 10;
+        margin-block-start: 0 !important;
+    }
```

Comment:
see - https://github.com/appwrite/console/blob/main/static/css/loading.css#L37C1-L44C2
that way, the appwrite icon doesn't shift.